Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -244,8 +244,10 @@ public NodejsProjectNode(NodejsProjectPackage package)
         }
 
         protected override void FinishProjectCreation(string sourceFolder, string destFolder) {
+            bool foundTypeScriptFile = false;
             foreach (MSBuild.ProjectItem item in this.BuildProject.Items) {
                 if (String.Equals(Path.GetExtension(item.EvaluatedInclude), NodejsConstants.TypeScriptExtension, StringComparison.OrdinalIgnoreCase)) {
+                    foundTypeScriptFile = true;
```

Comment:
Hmm this check looks like it'll capture both .d.ts and .ts files. Is that correct? If so we should try to limit it to just .ts